packages/web-runtime/src/App.vue

```diff
@@ -19,22 +19,23 @@
       </template>
       <div v-else key="core-content" class="uk-flex uk-flex-stretch">
         <transition :name="appNavigationAnimation">
-          <oc-sidebar
-            v-if="isSidebarVisible"
-            v-touch:swipe.left="handleNavSwipe"
-            class="oc-app-navigation"
-            :logo-img="logoImage"
-            :logo-alt="sidebarLogoAlt"
-            :nav-items="sidebarNavItems"
-            :class="sidebarClasses"
-            :fixed="isSidebarFixed"
-            :accessible-label="accessibleLabel"
-            @close="toggleAppNavigationVisibility"
-          >
-            <template v-if="sidebar.sidebarFooterContentComponent" v-slot:footer>
-              <component :is="sidebar.sidebarFooterContentComponent" />
-            </template>
-          </oc-sidebar>
+          <focus-trap v-if="isSidebarVisible" :active="windowWidth < 1200 && appNavigationVisible">
```

Comment:
Shouldn't the focus trap only be allowed to become active at all in the mobile full viewport overlay?

Reply:
Not sure... I would expect that if it opens it behaves kinda like a "modal" and traps the focus as well. No matter the resolution. Then if it is opened always (`windowWidth >= 1200`) no trap.

Reply:
We had a discussion with @marcus-herrmann on that, just not about the left but about the right sidebar. Quoting him:
```
Modal means "mode-changing", and the convention for modal is in my eyes, that other parts of the site (everything but the modal) is blocked from accessing). also visually it does not look like a modal at all.

what a modal typically does is to block all the other content to force a decision from the user. that is UI-wise not the case here.

the sidebar is imo rather a dialog (non-modal), if any
so focus should be able to leave (in the same vibe: all other contents, landmarks, headlines, links, etc not in the modal should be accessible even when the sidebar is present)
```
Same applies here. So from my point of view we can (and should) have a focus trap when the left or right sidebar respectively cover the whole viewport.

Reply:
I see. Ok, I'll adjust it then 👍